compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/OpOracle.cpp

```diff
@@ -100,6 +102,12 @@ ConstExprOpInfo ConstExprOpInfo::getForOp(Operation *op) {
     return {};
   }
 
+  // Target-dependent ops are not const-expr.
+  if (isa<IREE::LinalgExt::UpperBoundTileSizeOp, IREE::LinalgExt::SetEncodingOp,
+          IREE::Codegen::QueryTileSizesOp>(op)) {
```

Comment:
IIUC we dont need to add `QueryTileSizes` op here. In that case we wont need the dependence from `Codegen -> Utils`.